docs/topics/http/urls.txt

```diff
@@ -567,7 +567,7 @@ directions:
 
 The first one is the usage we've been discussing in the previous sections. The
 second one is what is known as *reverse resolution of URLs*, *reverse URL
-matching*, *reverse URL lookup*, or simply *URL reversing*.
+matching*, *reverse URL lookup*, or *URL reversing*.
```

Comment:
I think this can be reverted, it's highlighting the term that will be used in the rest of the page/documentation